clang-tools-extra/docs/clang-tidy/checks/bugprone/crtp-constructor-accessibility.rst

```diff
@@ -0,0 +1,82 @@
+.. title:: clang-tidy - bugprone-crtp-constructor-accessibility
+
+bugprone-crtp-constructor-accessibility
+=======================================
+
+Finds Curiously Recurring Template Pattern used in an error-prone way.
```

Comment:
"Detects error-prone CRTP usage, when the CRTP can be constructed outside itself and the derived class."

Sync with release notes.